std_msgs/Empty msg
---
int8 rate
